go.mod

```diff
@@ -2,4 +2,110 @@ module github.com/openshift/must-gather
 
 go 1.17
 
-require github.com/openshift/build-machinery-go v0.0.0-20210423112049-9415d7ebd33e
+require (
```

Comment:
Like mentioned before, I don't think we want to handle it here, it should be built outside of this repo and only installed here.

Reply:
That would require an RPM in the package repos. The tool is specific to the must gather data collection though. Also we want the tool to be always in sync with the MG and OCP versions. Building it here is actually much much easier and more reliable.

Reply:
Based on what I was told it is already built as RPM so that shouldn't be the problem.